GameDefinition/Content/ItemKind.hs

```diff
@@ -1617,13 +1618,13 @@ halberd2 = halberd
   { ifreq    = [("common item", 3 * 2), ("starting weapon", 1)]
   , iweight  = 4000
   , idamage  = 18 `d` 1
-  -- , idesc    = ""
+  , idesc    = "A long haft with a sharp blade."
```

Comment:
This and the next one are very rare and very powerful variants of the "war scythe". The first one is "improvised" on purpose to let the next ones be stronger. I see you made the third one really a halberd, which makes sense, but if so, let's change `iname` to `halberd` or `heavy halberd` (or to something else that fits the bill of heavy brutal polearm and that would differentiate it from the rather subtle unique artifact `Swiss Halberd` that is weaker, but has an extra effect; see, e.g., https://en.wikipedia.org/wiki/Pole_weapon).

Reply:
I've gone with halberd for the middle one and then bardiche for the last - like a halberd, but more choppy. Does that work?

Reply:
Sure. I'd perhaps somehow suggest the halberd is a real weapon, this time, not a make-shift one. But I'm often overdoing things, and your texts are proper sound-bites. :)

Reply:
What do you think (I don't insist at all; after all I have no clue how it will look in-game and click with the excitement of finding it for the first time after 100 games, using it, etc.).